Cargo.toml

```diff
@@ -142,21 +156,23 @@ rfd = "0.15"
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 winit = "0.30"
 web-sys = { version = "0.3.74", features = [
-    "Clipboard",
-    "ClipboardEvent",
-    "ClipboardItem",
-    "CompositionEvent",
-    "DataTransfer",
-    "Document",
-    "EventTarget",
-    "HtmlInputElement",
-    "InputEvent",
-    "KeyboardEvent",
-    "Navigator",
-    "TouchEvent",
-    "Window",
+  "Clipboard",
+  "ClipboardEvent",
+  "ClipboardItem",
+  "CompositionEvent",
+  "DataTransfer",
+  "Document",
+  "EventTarget",
+  "HtmlInputElement",
+  "InputEvent",
+  "KeyboardEvent",
+  "Navigator",
+  "TouchEvent",
+  "Window",
```

Comment:
People with different toml formatter settings keep diff-fighting over this one :D
I'll be happier if the changes don't include this :)

Reply:
Formatting changes reverted